lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -241,7 +241,7 @@ class FlutterConfiguration {
       _configuration?.canvasKitBaseUrl ?? _defaultCanvasKitBaseUrl;
   static const String _defaultCanvasKitBaseUrl = String.fromEnvironment(
     'FLUTTER_WEB_CANVASKIT_URL',
-    defaultValue: 'canvaskit/',
+    defaultValue: '/canvaskit/',
```

Comment:
I think this'd break the base-url functionality.

Should this be `/$BASE_URL/canvaskit/` ?

Reply:
We discussed this offline, I think this current behavior is a reasonable default and is consistent with what flutter.js is doing. In the future, we may want to deal with the base URI stuff in a more robust way. Our current behavior is not super consistent in general.